odl/set/sets.py

```diff
@@ -217,6 +237,15 @@ def __contains__(self, other):
         characters, `False` otherwise."""
         return isinstance(other, basestring) and len(other) == self.length
 
+    def contains_all(self, array):
+        """Test if `array` is an array of strings with correct length."""
```

Comment:
Because length is an integral part of a class instance? It's the same way NumPy string arrays are handled (length is part of the data type).